src/lib/TwaGenerator.ts

```diff
@@ -21,6 +21,7 @@ import {template} from 'lodash';
 import {promisify} from 'util';
 import {TwaManifest} from './TwaManifest';
 import Jimp = require('jimp');
+import Log from './Log';
```

Comment:
TypeScript question - why do some of these have `{}` and some don't?

Reply:
On the Log, we do `export default class Log`. on TwaManifest, only `export class TwaManifest`.